app/views/spree/admin/payments/source_views/_paypal.html.erb

```diff
@@ -15,21 +15,6 @@
       </dl>
     </div>
 
-    <% if payment.source.state != 'refunded' %>
-      <%= button_link_to Spree.t('actions.refund', :scope => :paypal), spree.paypal_refund_admin_order_payment_path(@order, payment), :icon => 'icon-dollar' %>
-    <% else %>
-      <div class="alpha six columns">
-        <dl>
-          <dt><%= Spree.t(:state, :scope => :paypal) %>:</dt>
-          <dd><%= payment.source.state.titleize %></dd>
-
-          <dt><%= Spree.t(:refunded_at, :scope => :paypal) %>:</dt>
-          <dd><%= pretty_time(payment.source.refunded_at) %></dd>
-
-          <dt><%= Spree.t(:refund_transaction_id, :scope => :paypal) %>:</dt>
-          <dd><%= payment.source.refund_transaction_id %></dd>
-        </dl>
-      </div>
-    <% end %>
+    <%= button_link_to Spree.t('actions.refund', :scope => :paypal), spree.paypal_refund_admin_order_payment_path(@order, payment), :icon => 'icon-dollar' %>
```

Comment:
Yeah, that will render a form page where you enter the amount to refund I think. The conditional branch just made sure you couldn't open the page if a previous refund was made.